runtime/src/common/sgx/seal.rs

```diff
@@ -30,17 +31,20 @@ pub fn seal(key_policy: Keypolicy, context: &[u8], data: &[u8]) -> Vec<u8> {
 ///
 /// # Panics
```

Comment:
The function doesn't panic anymore, so this heading and the description could be removed.